aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -1161,8 +1163,23 @@ module aptos_framework::stake {
             assume cur_validator_perf.successful_proposals + cur_validator_perf.failed_proposals <= MAX_U64;
         };
         let num_total_proposals = cur_validator_perf.successful_proposals + cur_validator_perf.failed_proposals;
-
-        let (rewards_rate, rewards_rate_denominator) = staking_config::get_reward_rate(staking_config);
+        let (rewards_rate, rewards_rate_denominator) = if (features::reward_rate_decrease_enabled()) {
+            let epoch_rewards_rate = staking_config::calculate_and_save_latest_epoch_rewards_rate();
+            if (fixed_point64::is_zero(epoch_rewards_rate)) {
```

Comment:
Why do we need to worry about this case?

Reply:
We divide by epoch_rewards_rate when calculating denominator in the following else branch.

Reply:
But how can epoch_rewards_rate be zero?

Reply:
Zero is a valid value for `rewards_rate`. We don't have this restriction so far. I think it's OK if the community really wants it.

Reply:
We could actually enforce rewards_rate > 0 esp now that we have a min rewards rate

Reply:
Can you elaborate why we should enforce rewards_rate > 0? It looks like a valid value for me. For example, in the future the community decides to remove inflation rewards and validators can only receive incentives from transaction fees.